src/resources/views/crud/inc/filters_navbar.blade.php

```diff
@@ -1,4 +1,4 @@
-<nav class="navbar navbar-expand-lg navbar-filters mb-0 pb-0 pt-0">
+<nav class="navbar navbar-expand-lg navbar-filters mb-1 pb-0 pt-0">
       {{-- Brand and toggle get grouped for better mobile display --}}
```

Comment:
For some reason, on this branch, the `simple` filter still works. The others... do not. When I click any of them... nothing happens for most. Except for `date_range` which does open, but it has a weird position on the screen:

 
![CleanShot 2023-02-26 at 10 37 34](https://user-images.githubusercontent.com/1032474/221400552-d02f4cc2-8a18-4e8f-bf61-9e9118b7fb96.gif)

I assume this is because the Dropdown HTML has changed, between Bootstrap v4 and Bootstrap v5? Or between CoreUI v2 and CoreUI v4? So I assume we're going to need to do some HTML or JS changes to ALL filters. If so, that's going to be a breaking change, so we need to do that on this branch, before we branch out and do a `tabler` theme.

Reply:
@tabacitu It requires, as you said, a change in HTML. It turns out in bootstrap 5 the dropdown changed an attribute from `data-toggle` to `data-bs-toggle`. This is a change in PRO branch...